NGitLab/Impl/ReleaseLinkClient.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using NGitLab.Models;
+
+namespace NGitLab.Impl
+{
+    // Handles interacting with Release links.
+    // Documentation: https://docs.gitlab.com/ee/api/releases/links
+    public class ReleaseLinkClient : IReleaseLinkClient
```

Comment:
The class doesn't need to be public as we only expose the interface

```suggestion
    internal sealed class ReleaseLinkClient : IReleaseLinkClient
```

Reply:
Changed to interal, and did the same for ReleaseClient